src/V3Builtin.cpp

```diff
@@ -0,0 +1,90 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: built-in packages and classes
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2004-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include "V3Ast.h"
+#include "V3Builtin.h"
+#include "V3FileLine.h"
+#include "V3Parse.h"
+#include "V3ParseSym.h"
+
+namespace V3Builtin {
+void parseStdPackage(V3Parse& parser) {
+    string filename = v3Global.opt.hierTopDataDir() + "/builtin.v";
+    std::ofstream* ofp = nullptr;
+    ofp = V3File::new_ofstream(filename);
+    if (!ofp->fail()) {
+        *ofp << R"(
+        package std;
+            class process;
+                integer x;
+                function new();
+                    x = 42;
+                endfunction;
+                typedef enum { FINISHED, RUNNING, WAITING, SUSPENDED, KILLED } state;
+                extern static function process self();
+                extern function state status();
+                extern function void kill();
+                extern task await();
+                extern function void suspend();
+                extern function void resume();
+            endclass
+        endpackage
+        )" << std::endl;
+    }
+    if (ofp) {
+        ofp->close();
+        delete ofp;
+    }
+    parser.parseFile(new FileLine(FileLine::commandLineFilename()), filename, false,
+                     "Cannot find file with built-in definitions: ");
+}
+
+void defineExterns(AstNetlist* rootp, V3ParseSym& parseSyms) {
+    auto* pkgp = rootp->modulesp();
+    while (pkgp->name() != "std") { pkgp = VN_CAST(pkgp->nextp(), NodeModule); }
+    parseSyms.pushNew(pkgp);
+    auto* classp = parseSyms.symCurrentp()->findIdFallback("process")->nodep();
+    auto* memberp = VN_CAST(classp, Class)->membersp();
+    while (memberp) {
```

Comment:
```suggestion
    for(auto* memberp = VN_CAST(classp, Class)->membersp(); memberp; memberp = memperp->nexpt()) {
```

But actually this code is broken as you can't assume members are only Classes, you need to iterate with it as a Node* then test with cast in the loop.